WordPress/openverse-frontend
# Images are now displays correctly on Home page in smaller devices

## Fixes
Fixes #[1209] by @[HILAYTRIVEDI]

## Testing Instructions
Open the open verse homepage and check-in the responsive device  Now all the images are visible perfectly.

## Checklist
